test/signature_test.js

```diff
@@ -3,32 +3,39 @@
 
 const assert = require('assert')
 const { before, describe, it } = require('mocha')
+const path = require('path')
 
-const { generatePublisherId, signBuffer, verifyBuffer } = require('../src/peer/identity')
+const { PublisherId, PrivateSigningKey, PublicSigningKey } = require('../src/peer/identity')
 const { makeSimpleStatement } = require('../src/metadata/statement')
 const { signStatement, verifyStatement } = require('../src/metadata/signatures')
 
+const CONCAT_PUBLISHER_ID_PUB58 = '4XTTM4JKrrBeAK6qXmo8FoKmT5RkfjeXfZrnWjJNw9fKvPnEs'
+const CONCAT_PUBLISHER_ID_PATH = path.join(__dirname, 'resources', 'publisher_ids', 'concat',
+  `${CONCAT_PUBLISHER_ID_PUB58}.privateKey`)
+const CONCAT_MESSAGE_FIXTURES = require('./resources/fixtures/concat-message-signature')
+const ETH_MESSAGE_FIXTURES = require('./resources/fixtures/ethereum-message-signature')
+
 describe('Signature verification', () => {
   let publisherId
 
```

Comment:
should add deterministic signature checking here as well, by analogy to my branch